### PR TITLE
TVMaze estimator

### PR DESCRIPTION
**Pull request by liiight**
TVMaze estimator for discover plugin

@gazpachoking Currently, some 'print' command are shown on manual series search.
This should be fixed shortly at pytvmaze side, so we can wait with merging this until that is fixed, or add another commit to suppress them.

Edit: let's sit on this for now. I'm working with pytvmaze to do something different and those changes will break behavior. I'll update when this is ready.

EDIT: @gazpachoking this is ready to be merged IMO. worked with pytvmaze dev and now it doesn't have those 'print' and is way more robust.

### CODE REVIEW

File: flexget/plugins/estimators/est_release_series_tvmaze.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        except ShowNotFound as e:
+            log.warning('Could not found show on TVMaze: %s' % e)
+            return
+        episode = tvmaze_show[season][episode_number]
+        if episode:
+            airdate = datetime.strptime(episode.airdate, '%Y-%m-%d')
+            log.debug('received airdate: {0}'.format(airdate))
+            return airdate
+        else:
+            log.debug('No episode info obtained from TVMaze for %s season %s episode %s' % (
```
Comment (cvium) [inline flexget/plugins/estimators/est_release_series_tvmaze.py]: I think we should try to be consistent. Either use the modulo operator or use .format :D

Comment (liiight) [inline flexget/plugins/estimators/est_release_series_tvmaze.py]: an oversight, sorry.

Comment (cvium) [inline flexget/plugins/estimators/est_release_series_tvmaze.py]: It's also at line 51 :P

I'm just being picky.

Comment (liiight) [inline flexget/plugins/estimators/est_release_series_tvmaze.py]: thanks god commits are free :)

**Pull request merged**